cli/in.go

```diff
@@ -53,7 +53,7 @@ func (i *InStream) SetRawTerminal() (err error) {
 // RestoreTerminal restores normal mode to the terminal
 func (i *InStream) RestoreTerminal() {
 	if i.state != nil {
-		term.RestoreTerminal(i.fd, i.state)
+		term.RestoreTerminal(i.fd, i.state) // nolint
```

Comment:
If it's about ignoring an error, it's probably better to just be explicit that we don't care; i.e.:  `_ := term.RestoreTerminal(...)`.